src/org/robockets/steamworks/RobotMap.java

```diff
@@ -19,6 +17,8 @@
     public static TalonSRX conveyorSpeedControllerOne = new TalonSRX(PortNumbers.SHOOTER_CONVEYOR_SC_ONE);
     public static TalonSRX conveyorSpeedControllerTwo = new TalonSRX(PortNumbers.SHOOTER_CONVEYOR_SC_TWO);
 
+    public static ADXRS450_Gyro gyro = new ADXRS450_Gyro(SPI.Port.kOnboardCS0);
```

Comment:
To keep everything in one place. That's the whole point of `PortNumbers` and `RobotMap`, isn't it?

`PortNumbers` would have an `SPI.Port` named `gyroPort`.